### PR TITLE
NF: comment model/deck/deckconfig

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This comes from #8817 from @TarekkMA

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]
Lgtm

File: AnkiDroid/src/main/java/com/ichi2/libanki/Model.java [hunk 1]
```diff
@@ -40,23 +39,36 @@
...
     public Model(String json) {
         super(json);
     }
 
     @Override
     @CheckResult
     public Model deepClone() {
-        Model clone = new Model();
-        return deepClonedInto(clone);
+        return new Model(this);
```
Comment (TarekkMA) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Model.java:71]: Why `Model` is not using `deepClonedInto(clone);` like `Deck` and `DeckConfig`?

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/Model.java:71]: Oh right, this should not have been changed here at all. It was only supposed to introduce comment...

**Review by david-allison-1 — APPROVED** [review]
Lgtm

**Review by mikehardy — APPROVED** [review]

**Pull request merged**